pkg/kubelet/kubelet.go

```diff
@@ -190,10 +193,15 @@ const (
 	eventedPlegMaxStreamRetries = 5
 
 	// backOffPeriod is the period to back off when pod syncing results in an
-	// error. It is also used as the base period for the exponential backoff
-	// container restarts and image pulls.
+	// error.
 	backOffPeriod = time.Second * 10
 
+	// base period for the exponential backoff for container restarts.
```

Comment:
nit: s/base/initial/ ?

Also, either start the comment with `containerBackOffPeriod` or capitalize it.

Reply:
Fixed in 85b3fd5